game/templates/game/board.html

```
@@ -66,7 +66,7 @@
             <div style="margin-bottom: 20px; color:#f0c040;">
                 <a href="/" class="back-btn">← Back to Home</a>
             </div>
-            <div class="promo-title"
+            <div class="promo-title"git sta
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)